compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/CompilerUtils.java

```diff
@@ -69,4 +73,13 @@ public static String getPackageIDStringWithMajorVersion(PackageID packageID) {
         return org + packageID.name + Names.VERSION_SEPARATOR.value + getMajorVersion(packageID.version.value);
     }
 
+    public static boolean isFunctionParameter(BSymbol symbol, List<BLangSimpleVariable> params) {
```

Comment:
While we are passing params here, this is just a check to see if the symbol belongs to a variable in the list. I would rename the method appropriately.

Reply:
And variable/param names.